modules/administration/pages/maintenance-window.adoc

```diff
@@ -86,3 +92,16 @@ zypper migration
 ----
 
 This procedure is also known as a Service Pack (SP) migration.
+
+
+
+== Inter-Server Synchronization Slave Server
+
+Inter-server synchronization slave servers should be updated as soon as
```

Comment:
```suggestion
Inter-server synchronization Slave Servers should be updated as soon as
```

Reply:
@Loquacity this is a corner case.  Please help!

Reply:
I don't see any reason to initcap here. It's not a proper noun.

Reply:
> I don't see any reason to initcap here. It's not a proper noun.

Reply:
Server is short of `{productname} Server`, don't we initcap that?

Also, ISS Masters and Slaves are initcapped everywhere in the UI:

![Screenshot from 2019-10-03 09-32-06](https://user-images.githubusercontent.com/250541/66107562-dade6400-e5c0-11e9-95c2-60803814f69c.png)

and API doc:

![Screenshot from 2019-10-03 09-32-52](https://user-images.githubusercontent.com/250541/66107572-e2057200-e5c0-11e9-808e-c4e70175a091.png)


...so I thought we should stay consistent.

Reply:
ok, this again is a @Loquacity remark.  Of course, I vote to be consistent as possible.

Reply:
I would argue that the UI is wrong. Basically, initcaps make text harder to read, so we should avoid them wherever possible. Also, the argument that "server" is short for "{productname} Server" is silly, because grammar doesn't work like that. "Server" on its own is a noun, and adding the {productname} in makes it a proper noun. SUSE does not own the word "Server", they own the noun phrase "SUSE Manager Server".